src/Printer.hs

```diff
@@ -0,0 +1,40 @@
+module Printer where
+
+import Data.Tree
+import Data.Maybe
+import Data.Text.Prettyprint.Doc
+
+import Game hiding (Tree(..))
+import Utils
+
+instance Pretty Cell where
+    pretty X = pretty 'x' 
+    pretty O = pretty 'o'
+    pretty B = pretty '.'
+
+instance Pretty Player where
+    pretty PlayerX = pretty ("Player X" :: String)
```

Comment:
I presume if I remove `OverloadedStrings` I wouldn't need the type annotation here?

Reply:
Yep